docs/dietpi_tools.md

```diff
@@ -127,163 +129,186 @@ You can return to the **DietPi-Software** tool to make further changes at any ti
 
 ## Configuration
 
-`dietpi-config` {: #DietPi-Config }
+=== "DietPi configuration"
 
-: Configure various system settings, from display / audio / network to _auto start_ options.
+    Configure various system settings, from display / audio / network to _auto start_ options.  
+    Run `dietpi-config`.
 
-: ![DietPi-Config screenshot](assets/images/dietpi-config.jpg)
```

Comment:
Great, I think this allows us to get rid of the definition list extension: https://github.com/MichaIng/DietPi-Docs/blob/master/mkdocs.yml#L33
I don't see a reasonable use case for this, as it does nothing more then indenting everything below the definition name. But I personally prefer a clear header and no indentation for the content, or if course the tabs that you implemented now as discussed.